core/storage/suggestion/gae_models.py

```diff
@@ -468,6 +468,33 @@ def get_in_review_suggestions_in_score_categories(
             cls.author_id != user_id
         )).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
 
+    @classmethod
+    def get_in_review_translation_suggestions(
+        cls,
+        user_id: str,
+        language_codes: List[str]
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Fetches all translation suggestions that are in-review where the
+        author_id != user_id and language_code matches one of the supplied
+        language_codes.
+
+        Args:
+            user_id: str. The id of the user trying to make this query. As a
+                user cannot review their own suggestions, suggestions authored
+                by the user will be excluded.
+            language_codes: list(str). List of language codes that the
+                suggestions should match.
+
+        Returns:
+            list(SuggestionModel). A list of the matching suggestions.
+        """
+        return cls.get_all().filter(datastore_services.all_of(
+            cls.status == STATUS_IN_REVIEW,
+            cls.suggestion_type == feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            cls.author_id != user_id,
+            cls.language_code.IN(language_codes)
```

Comment:
I think we should still test it on the backup server. Please submit a testing request.

Reply:
FYI I think you'll need a new index for subsets of an existing indexes -- I seem to remember we ran into a case before where we thought it would be handled by App Engine, but it wasn't.

Reply:
Submitted a testing request.